streamtube.js

```diff
@@ -34,7 +39,7 @@ var streamToTube = function(stream, maxDivergence, minDistance) {
 		if (maxDivergence === 0) {
 			r = minDistance * 0.05;
 		}
-		currentIntensity = vec3.length(fwd);
+		currentIntensity = vec3.length(fwd) / maxNorm;
```

Comment:
I was going to suggest dealing with division by zero here, but on second thought, it's not necessary. With zero field, you'd end up with a division by zero. But with a zero field you wouldn't have a stream anyway, so you shouldn't encounter this line with zero maxNorm.